src/main/java/bookstore/dto/BookRequestDto.java

```diff
@@ -1,13 +1,19 @@
 package bookstore.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class BookRequestDto {
+    @NotNull
     private String title;
+    @NotNull
     private String author;
+    @NotNull
     private String isbn;
+    @Min(0)
```

Comment:
Min won't work for BigDecimal. Use @Positive or @DecimalMin

Reply:
Not fixed